base/array.jl

```diff
@@ -363,7 +363,7 @@ function zeros end
     ones([T=Float64,] dims...)
 
 Create an `Array`, with element type `T`, of all ones with size specified by `dims`.
-See also [`zeros`](@ref), [`similar`](@ref).
+See also: [`fill`](@ref), [`zeros`](@ref).
```

Comment:
Similarly here, why ref `fill` and `zeros` (rather than `fill` and `similar`)?

Reply:
See above.